### DISCUSSION TITLE
Some audio not playing

### DISCUSSION THREAD

**Discussion by AStoker** [MiczFlor/RPi-Jukebox-RFID#2208] [Q&A]
Trying to figure out what’s going on. I have some mp3s that play fine, and some flac files that play fine. But every once in a while I’ll have a file that doesn’t play. Are there some specified codecs that need to be used for the files that maybe I’m not using? Can’t find details on that anywhere.

**Comment by s-martin** [comment]
Usually no special codecs etc need to be installed.

Which version do you use and do you use any Spotify modifications?

Phoniebox uses [MPD](https://www.musicpd.org/) under the hood.

If you are using Version 2.x you could check the logs with `tail -n 200 /var/log/syslog` and post the logs here.

**Reply by AStoker** [reply]
I'm using 2.5.0 - 4101984b. No Spotify is set up, don't use it.
Regarding the logs, thanks, that's helpful. Here's what I get 
```
Jan  6 20:08:37 jukepi mpd: player: played "NEEDTOBREATH/Favorites/04. Dreams.flac"
Jan  6 20:08:37 jukepi mpd: exception: No such song
Jan  6 20:08:37 jukepi lighttpd[93753]: error adding file:///home/pi/RPi-Jukebox-RFID/shared/audiofolders/Fun Music/Hello - Martin Solveig and Dragonette.mp3: No such song
```
The file is definitely there. When I use SMB to navigate the audio files, I can even play it on my computer. Are there file naming conventions that need to be followed?

**Reply by AStoker** [reply]
Actually, that appears to be the issue. Renaming the file to get rid of the dash seemed to resolve it. Seems weird that a dash would throw this mess though...

**Reply by s-martin** [reply]
That's weird and shouldn't be the case.
We need to investigate this.

Could you try to rename a previously working file, if it does work after renaming it with a dash?

I opened an issue #2210 
Please follow up there
